hack/non-olm-install/install-gitops-operator.sh

```diff
@@ -0,0 +1,254 @@
+}
+
+function init_work_directory() {
+  # create a temporary directory and do all the operations inside the directory.
+  TEMP_DIR=$(mktemp -d "${TMPDIR:-"/tmp"}/gitops-operator-install-XXXXXXX")
+  echo "Using temp directory $TEMP_DIR"
+  # cleanup the temporary directory irrespective of whether the script ran successfully or failed with an error.
+  trap cleanup EXIT
+}
+
+# Checks if the pre-requisite binaries are already present in the PATH,
+# if not installs appropriate versions of them.
+# This function just checks if the binary is found in the PATH and 
+# does not validate if the version of the binary matches the minimum required version.
+function check_and_install_prerequisites {
+  # install kustomize in the the temp directory if its not available in the PATH
+  KUSTOMIZE=$(which kustomize)
+  install_kustomize
+
+  # install kubectl in the the temp directory if its not available in the PATH
+  KUBECTL=$(which kubectl)
+  install_kubectl
+
+  # install yq in the the temp directory if its not available in the PATH
+  YQ=$(which yq)
+  install_yq
+}
+
+# Checks if the openshift-gitops-operator is already installed in the system.
+# if so, stores the previous version which would be used for rollback in case of
+# a failure during installation.
+function get_prev_operator_image() {
+  PREV_OPERATOR_IMG=$(${KUBECTL} get deploy/gitops-operator-controller-manager -n ${NAMESPACE_PREFIX}system -o jsonpath='{..image}');
+  echo "PREV OPERATOR IMAGE : ${PREV_OPERATOR_IMG}"
+}
+
+# Prepares the kustomization.yaml file in the TEMP_DIR which would be used 
+# for the installation.
+function prepare_kustomize_files() {
+  # create the required yaml files for the kustomize based install.
+  create_kustomization_init_file
+  DOCKER=$(which ${DOCKER})
+  if [[ ${USE_BUNDLE_IMG} == "true" && ! -z "${DOCKER}" ]]; then
+    echo "Generating env-overrides.yaml file from the CSV defined in the bundle image"
+    create_deployment_patch_from_bundle_image
+  else
+    echo "Bundle image is disabled or Docker binary ${DOCKER} not found in PATH"
+    echo "Generating env-overrides.yaml file from the values provided in the environment variable"
+    create_image_overrides_patch_file
+  fi
+}
+
+# Code execution starts here
+# Get the options
+while getopts ":iu" option; do
+  case $option in
+    i) # use kubectl binary to apply the manifests from the directory containing the kustomization.yaml file.
+      echo "installing ..."
+      init_work_directory
+      check_and_install_prerequisites
+      get_prev_operator_image
+      prepare_kustomize_files
+      ${KUBECTL} apply -k ${TEMP_DIR}
+      # TODO: Remove the workaround of adding RBAC policies once the below issue is resolved.
+      # Workaround for fixing the issue https://github.com/redhat-developer/gitops-operator/issues/148
+      ${KUBECTL} apply -f https://raw.githubusercontent.com/redhat-developer/gitops-operator/master/hack/non-bundle-install/rbac-patch.yaml
```

Comment:
Looks like this line is trying to apply a file with the assumption that the PR is already merged into master, which is why it errors out when I try to execute it as a part of testing the PR 
could we have it try to apply it from a local relative path instead? 
@anandf

Reply:
Yes. I put the path under the assumption that it would be valid when the changes gets merged to `master` branch.
I wanted the install script to be run with a simple `wget` or `curl` command without having the need to use `git clone`. I am thinking of removing this line and moving it to the `README` file as a note to have solution to this known issue.

Reply:
I have removed this line in my latest commit and have provided documentation on how to apply this patch in `README.md` file. under the known issues and workarounds section.